oeis.py

```diff
@@ -261,6 +260,45 @@ def A001220(start: int = 0, limit: int = 20) -> Collection[int]:
     return sequence
 
 
+@oeis
+def A008587(start: int = 0, limit: int = 20) -> Collection[int]:
+    "Multiples of 5."
+    sequence = []
+
+    for i in range(start, limit):
+        s = str(i)
+        if s[len(s) - 1] == "5" or s[len(s) - 1] == "0":
```

Comment:
`len(s) - 1` is equivalent to `-1`, you should simplify this one.

Also instead of using a string, maybe just try using a modulo (`%`), like `if i % 5 == 0`.

Also I bet if I as a limit of 10 I won't get 10 numbers, which is wrong.